src/logic/safe/store/middleware/notificationsMiddleware.ts

```diff
@@ -67,7 +67,7 @@
~  dispatch(push(`/safes/${safeAddress}/transactions`))
~}
~
~const notificationsMiddleware = (store) => (next) => async (action) => {
~  const handledAction = next(action)
```

Comment:
You shouldn't need the ternary anymore, it should be always defined

Reply:
Checked and changed